.github/workflows/pull-request-automation.yml

```diff
@@ -6,6 +6,7 @@ name: Pull request automation
 
 jobs:
     pull-request-automation:
+        timeout-minutes: 10
```

Comment:
Does this ever reach this limit? I'm not sure that I have ever seen this workflow take more than 10 seconds.

Reply:
This would have been helpful during the last GitHub Actions outage; this process in particular was running for hours in some cases. Now, I'm not certain that GHA would actually timeout the process in the context of an outage, but I think it's worth a try.

Reply:
An example of this happening; this particular job [ran for 1h 2m](https://github.com/WordPress/gutenberg/actions/runs/857494540), on the day of a [big GitHub Actions outage](https://www.githubstatus.com/incidents/m16jzl31gnqt).

Reply:
Got it! Seems like a good idea. If GHA is having an outage, I wonder if that limit would even work? I guess it depends on the outage itself.

Another option could be to add the `concurrency` option to this workflow. That would just cancel all other preceding workflows for the same branch. I don't know how the code is structured within `packages/project-management-automation` though. So I can't say whether it's essential that it runs for every `push` event.